src/core/util/util.js

```diff
@@ -83,12 +83,26 @@ function UtilFactory($document, $timeout, $compile, $rootScope, $$mdAnimate, $in
       return $options.getOption ? $options.getOption(optionName) : $options[optionName];
     },
 
+    isRtl: function(attrs) {
+      var dir = angular.isDefined(attrs) && attrs.hasOwnProperty('dir') && attrs.dir;
+
+      switch (dir) {
+        case 'ltr':
+          return false;
+        
+        case 'rtl':
+          return true;
+      }
+      
+      return ($document[0].dir == 'rtl' || $document[0].body.dir == 'rtl');
+    },
+
     /**
      * Bi-directional accessor/mutator used to easily update an element's
      * property based on the current 'dir'ectional value.
      */
-    bidi : function(element, property, lValue, rValue) {
-      var ltr = !($document[0].dir == 'rtl' || $document[0].body.dir == 'rtl');
+    bidi: function(element, property, lValue, rValue) {
```

Comment:
Is this still used anywhere after the changes in this PR?

Reply:
Yes, it is used three times in `sticky` component for CSS transformations [here](https://github.com/marosoft/material/blob/wip/rtl-unification/src/components/sticky/sticky.js)